src/providers/ldap/sdap_async_nested_groups.c

```diff
@@ -1495,6 +1498,19 @@ sdap_nested_group_single_step_process(struct tevent_req *subreq)
             }
         }
 
+        /* The original DN of the user object itself might differ from the one
+         * used inthe member attribute, e.g. different case. To make sure if
```

Comment:
Just a minor failure but "inthe" should be separated with a space.

Reply:
Thanks, I've updated my branch https://github.com/sumit-bose/sssd/commits/ghost_hash_dn .